docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -77,3 +77,16 @@ We are gradually rolling out our new browse dashboards user interface. With this
 To learn more, refer to the following video demo.
 
 {{< video-embed src="/media/docs/grafana/2023-09-11-New-Browse-Dashboards-Enablement-Video.mp4" >}}
+
+## Temporary Credentials in CloudWatch Data Source
+
+<!-- Michael Mandrus, Ida Štambuk, Sarah Zinger  -->
+<!-- Cloud -->
+
+_Available in private preview in Grafana Cloud_
+
+`Grafana Assume Role` lets Grafana Cloud users of the CloudWatch Data Source authenticate with AWS without having to create and maintain long term AWS Users. By using the new Grafana Assume Role authentication method, you will no longer have to rotate access and secret keys in your CloudWatch Datasource. Instead Grafana Cloud customers can create an IAM Role with atrust relationship with Grafana's AWS Account and Grafana's AWS Account will then use STS to create temporary credentials to access the customer's AWS data.
```

Comment:
Realizing I didn't respond to this comment, sorry!

Spelled out STS, although like many aws acronyms it's hard to say if people are more familiar with the acronym or what it stands for haha. I think people who concern themselves with aws and auth will be pretty familar it's not that unique to grafana or anything. 

I'm not sure I totally follow the second question. 
Maybe this helps?

There's always been an assume role option available that looks like this: 
<img width="758" alt="Screenshot 2023-09-27 at 3 25 03 PM" src="https://github.com/grafana/grafana/assets/6620164/de2c044f-ea27-4862-b0c9-96525f35ad44">

Users have been able to have their primary aws credentials assume a role for a long time. I think this was primarily used in OSS and On-Prem environments and mostly using the aws-sdk which would allow them to attach credentials directly to whatever machine was running Grafana (like EC2). That said it has also possible to use Assume role in cloud, they would just also need to provide secret and access keys first:
<img width="740" alt="Screenshot 2023-09-27 at 3 27 20 PM" src="https://github.com/grafana/grafana/assets/6620164/9051d87b-3bf8-4305-b97c-e38286ac8d98">
which kind of takes away from some of the main benefits of using an iam role in the first place.

So now we have a new feature that we're calling Grafana Assume Role, that lets Grafana Cloud users specify an ARN without giving us secret keys (kind of like the AWS SDK Default option above)
<img width="631" alt="Screenshot 2023-09-27 at 3 24 02 PM" src="https://github.com/grafana/grafana/assets/6620164/34b21e9f-def5-43a4-b734-8b524a0feebb">

The ARN field doesn't change but the authentication provider does. 

I know this is super confusing because it's all called arn assume role haha, so I'm trying to make it more distinct/understandable

Reply:
Thanks for the clarification! To my mind, based on your explanation and our style guidelines, the term should just be  plain text except when we're referencing it specifically in the UI (then it's bold and capitalized as it is in the UI; which I've done in one instance in the docs). As for the capitalization, I'm iffy. @lwandz13, as this is your area, I'll leave the final decision on that to you, but I've made suggestions for it to remove the code formatting on the term in the PR and you can take or leave those suggestions as you see fit.